tensorflow/src/iosX64Main/kotlin/dev.icerock.moko.tensorflow/Interpreter.kt

```diff
@@ -113,6 +113,10 @@ actual class Interpreter(
                 TensorDataType.INT32 -> IntArray(outputTensor.dataType.byteSize()) // Fixme:
                 TensorDataType.UINT8 -> UIntArray(outputTensor.dataType.byteSize()) // Fixme:
                 TensorDataType.INT64 -> LongArray(outputTensor.dataType.byteSize()) // Fixme:
+                TensorDataType.STRING -> TODO()
```

Comment:
if we not have `TensorDataType.STRING` implementation so maybe just remove this case?

Reply:
It's in org.tensorflow.lite.DataType class, so it's required now to have String type(in android)
Or we can throw NotImplementedException in  dev.icerock.moko.tensorflow.toTensorDataType() which is android-only sourceSet